lxd/main_forksyscallgo.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
It would be useful to add a comment to this file explaining why it exists compared to the existing forksyscall command.

Reply:
Have added! Thanks!